soroban-env-host/src/native_contract/token/asset_info.rs

```diff
@@ -0,0 +1,25 @@
+use crate::native_contract::token::public_types::AssetInfo;
+use crate::native_contract::token::storage_types::DataKey;
+use crate::{host::Host, HostError};
+use soroban_env_common::{Env, TryIntoVal};
+
+// Metering: *mostly* covered by components.
```

Comment:
Is this actually true? This is just a storage read, probably should be covered completely.

Reply:
Good point, I missed this when I renamed the file. Removed.